portfolio/src/main/webapp/script.js

```diff
@@ -97,25 +98,25 @@ function createListElement(key, text) {
 
 // Check if the user is logged in or not, only display the comment function
 // if they are logged in.
-var xhttp = new XMLHttpRequest();
+const xhttp = new XMLHttpRequest();
 xhttp.onreadystatechange = function() {
   if (this.readyState == 4 && this.status == 200) {
     commentTitle = document.getElementById('comment-header');
     if (xhttp.responseText.includes('<p>You are logged in!</p>')) {
-      var form = document.createElement('form');
+      const form = document.createElement('form');
       form.setAttribute('method', 'POST');
       form.setAttribute('action', '/data');
 
-      var text = document.createElement('p');
+      const text = document.createElement('p');
       text.innerText = 'Enter your feedback here:';
 
-      var textArea = document.createElement('textarea');
+      const textArea = document.createElement('textarea');
       textArea.name = 'text-input';
       textArea.required = '';
 
-      var br = document.createElement('br');
+      const br = document.createElement('br');
```

Comment:
The name _br_ does not help me understand why we need this.

Reply:
renamed to breakElement